resources/assets/js/stories/JobPosterBuilder/Intro.stories.tsx

```diff
@@ -1,14 +1,47 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
 import { withIntl } from "storybook-addon-intl";
 import { action } from "@storybook/addon-actions";
-import JobBuilderIntro from "../../components/JobBuilderIntro/JobBuilderIntro";
+import fakeJob from "../../fakeData/fakeJob";
+import IntroForm from "../../components/JobBuilderIntro/IntroForm";
+import { Job } from "../../models/types";
 
-const stories = storiesOf("Job Poster Builder|Intro", module).addDecorator(
-  withIntl,
-);
+const stories = storiesOf("Job Poster Builder|Intro", module)
+  .addDecorator(withInfo)
+  .addDecorator(withIntl);
 
-stories.add(
-  "Body",
-  (): React.ReactElement => <JobBuilderIntro handleSubmit={action("Submit")} />,
-);
+const handleSubmit = async (): Promise<Job> => {
+  action("Submit")();
+  return fakeJob();
+};
+
+stories
+  .add(
+    "Intro Body",
```

Comment:
The story has a heading of `Intro` on line 10 so this is a little redundant. Maybe "New Job" and "Existing Job" like in the details story?